src/pytest_cov/plugin.py

```diff
@@ -59,6 +59,9 @@ def pytest_configure(config):
     """Activate coverage plugin if appropriate."""
     if config.getvalue('cov_source'):
         if not config.pluginmanager.hasplugin('_cov'):
+            if not config.option.cov_report:
```

Comment:
Is this check really necessary? I'm curious in what situation this would be hit.